src/Razor/benchmarks/Microsoft.AspNetCore.Razor.Microbenchmarks/LanguageServer/RazorLanguageServerBenchmarkBase.cs

```diff
@@ -125,17 +124,12 @@ public void Dispose()
 
     internal class NoopLogger : ILogger, ILspLogger
     {
-        public IDisposable BeginScope<TState>(TState state)
-        {
-            throw new NotImplementedException();
-        }
-
         public bool IsEnabled(LogLevel logLevel)
         {
             return true;
         }
 
-        public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+        public void Log<TState>(LogLevel logLevel, TState state, Exception exception, Func<TState, Exception, string> formatter)
```

Comment:
Can `state` be removed here?

Reply:
This is how the message gets passed currently. Will fix in a follow up PR.